tests/io_performance/image/run-iostat.sh

```diff
@@ -5,5 +5,5 @@
 #Running IOSTAT
 for (( i = 1; i <= "$IOSTAT_COUNT"; i++ ))
 do
-iostat -xm 3 "$COUNT_ITER" > ~/"$GIT_REPO"/"$FOLDERNAME"/Configs/Test-results/Iostat/iostat-fio-group-"$i"
```

Comment:
$FOLDERNAME instead of GIT_REPO is not reflected in README
Also FOLDERNAME is not a good name for anything.

Reply:
$FOLDERNAME is an internal env variable that is responsible for the name of the directory that will be added to the repository. It is set either from the name of the branch or automatically. It is not related to $GIT_REPO and does not need to be displayed in the README file. $GIT_REPO is present in the README file.